example/handwritten-src/swift/TXSTextboxListenerImpl.swift

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
Given this is acting as sample code, I don't like that there's now no example in use of ObjC implementing a Djinni class.  TXSTextboxListenerImpl.h is still around but not used.
Could you expand the example to demonstrate both?  E.g. have two different impl types used at different times, or where one calls the other, or with a run-time condition to pick which one gets used (so that both codepaths are compiled, even if not run).